pkgs/top-level/all-packages.nix

```diff
@@ -13616,7 +13616,10 @@ with pkgs;
 
   mozart2 = callPackage ../development/compilers/mozart {
     emacs = emacs-nox;
-    jre_headless = jre8_headless; # TODO: remove override https://github.com/NixOS/nixpkgs/pull/89731
+  };
+
+  mozart2WithGui = callPackage ../development/compilers/mozart {
+    installDesktop = true;
   };
```

Comment:
I'm not entirely sure on this, but I think we should expose two attributes here like this:

```nix
{

  mozart2-unwrapped = callPackage ../development/compilers/mozart/unwrapped.nix { };  

  mozart2 = callPackage ../development/compilers/mozart { };

}
```

This way the default is to have `mozart2` be wrapped with emacs, but for people who don't want that there's `mozart2-unwrapped`.

Reply:
Ok